public/components/common/modules/events-enhance-discover-fields.ts

```diff
@@ -106,7 +106,14 @@ export const EventsEnhanceDiscoverCell = {
   }), {
     contentRegex: /(\w+)/g,
     element: 'span'
-  })
+  }),
+  'syscheck.value_name': (content, rowData, element, options) => {
+    if (content) return;
+    const container = document.createElement("tr");
```

Comment:
why is the `container` an `tr` HTML element?

Reply:
It was meant to be a `td` element to match the rest of the cells in the table. Fixed in [97821ec](https://github.com/wazuh/wazuh-kibana-app/pull/3279/commits/97821ec47caf3d8512f3928f093c2240bcb0e59c)